py/objmodule.h

```diff
@@ -35,10 +35,20 @@
 #endif
 
 extern const mp_map_t mp_builtin_module_map;
-extern const mp_map_t mp_builtin_extensible_module_map;
 
-mp_obj_t mp_module_get_builtin(qstr module_name, bool extensible);
+#if MICROPY_HAVE_REGISTERED_EXTENSIBLE_MODULES
+extern const mp_map_t mp_builtin_extensible_module_map;
+#endif
 
+#if MICROPY_HAVE_REGISTERED_EXTENSIBLE_MODULES
```

Comment:
Could probably combine this and the above `#if` into one `#if` block.

Reply:
done.